docs/PROMETHEUS_METRICS.md

```diff
@@ -0,0 +1,136 @@
+# Prometheus-Compatible Metrics in EVE
+
+This document describes how EVE exposes system metrics in the Prometheus exposition format.
+
+## Node Exporter Metrics
+
+EVE exposes a broad set of metrics under the `node_exporter` style.
+From the EVE itself, you can access these metrics at:
+
+```text
+https://localhost:9100/metrics
```

Comment:
Doesn't this mean that the port number 9100 should be added as a rule in pkg/pillar/dpcreconciler/linux.go so that non-local access is blocked?

Also, it would make sense to run a full nmap to check if there are any open TCP or UDP ports which can be accessed from the Ethernet.

Reply:
Thank you for bringing this to my attention. I will review the code of the DPCReconciler.

Reply:
I've added the iptables rules to lock non-local access.
I'll try to scan the device to prove it works.

Reply:
Argh. The rules I added also break accessibility to the metrics from another container... Will have to fic it now.